importer/analytics-datastore-elastic-search/docker-compose.config.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.9'
+
+services:
+  disi-es-index-importer:
+    image: curlimages/curl:7.77.0
+    deploy:
+      restart_policy:
+        condition: none
+    configs:
+      - source: disi-es-index-config-fhir-report-patient.json
+        target: /fhir-report-patient.json
+    command:
+      [
+        'sh',
+        '-c',
+        "until nc -vz analytics-datastore-elastic-search 9200; do echo ES not running yet; sleep 1; done; curl -XPUT --fail-with-body 'http://analytics-datastore-elastic-search:9200/fhir-report-patient' -u elastic:${ES_ELASTIC-dev_password_only} -H 'Content-Type: application/json' -d@/fhir-report-patient.json"
```

Comment:
You must think carefully about which of these steps should continue if the previous step fails. If you do `first_command; second_command` with `first_command` failing, second command will continue to run.

But, if you do `first_command && second_command`, with `first_command` failing, then `second_command` will not run.

Reply:
Please address this in the other config importers too.

Reply:
I think it was made like that to loop waiting for elastic to start, if elastic responds, it will run the request. 
it may be an infinite loop though ... 

So fixed it to wait for 60s and it will exit 1 if it fails to connect to elastic [1439fbd](https://github.com/jembi/disi-on-platform/pull/22/commits/1439fbdee70057331dab65fda986bfd2c79b4a37)
your thoughts?